assets/scripts/admin/topbar.js

```diff
@@ -0,0 +1,42 @@
+$(document).on('ready pjax:scriptcomplete', function () {
+    function makeTopbarSticky() {
+        const topbar = $('.menubar:not(.surveymanagerbar)');
+        if (topbar.length == 0) {
+            return;
+        }
+        const topbarOffset = topbar.offset().top;
+
+        // When the topbar is floated, the space it occupies is released, so the rest of the content "jumps" up.
```

Comment:
Nice with comments. :+1:

Reply:
I like comments :) Sometimes more important than code.
Code reflects what I think is the solution.
Comment tells what I think is the solution.
If code fails, comments allow to still see which was the intention